src/components/Datepicker/Datepicker.tsx

```diff
@@ -71,21 +71,23 @@ export interface FlowbiteDatepickerPopupTheme {
   };
 }
 
-export interface DatepickerProps extends Omit<TextInputProps, 'theme'> {
+export interface DatepickerProps extends Omit<TextInputProps, 'theme' | 'defaultValue' | 'value'> {
   open?: boolean;
   inline?: boolean;
   autoHide?: boolean;
   showClearButton?: boolean;
   labelClearButton?: string;
   showTodayButton?: boolean;
   labelTodayButton?: string;
-  defaultDate?: Date;
+  defaultValue?: Date;
```

Comment:
I'm thinking if we should mimic the types of `value` prop, which are `Date | null` 🤔